pkg/scheduler/framework/plugins/selectorspread/selector_spread.go

```diff
@@ -181,8 +181,7 @@ func (pl *SelectorSpread) PreScore(ctx context.Context, cycleState *framework.Cy
 	if skipSelectorSpread(pod) {
 		return nil
 	}
-	var selector labels.Selector
-	selector = helper.DefaultSelector(
+	var selector = helper.DefaultSelector(
```

Comment:
```suggestion
	selector := helper.DefaultSelector(
```

Reply:
+1

Reply:
Thanks for the suggestion, i have changed this.